parameter/FloatingPointParameterType.cpp

```diff
@@ -252,9 +255,7 @@ bool CFloatingPointParameterType::toBlackboard(
 
     // Cast is fine because dValue has been checked against the value range
     float fValue = static_cast<float>(dUserValue);
-    // Using an intermediary reference variable to avoid klocwork false positive
-    const uint32_t &value = reinterpret_cast<const uint32_t &>(fValue);
-    uiValue = value;
+    utility::binaryCopy(fValue, uiValue);
```

Comment:
Lets pretend that binary returns the copy:
~~~cpp
auto fValue = utility::binaryCopy<float>(dUserValue);
~~~
I prefer to initialize variable on declaration.